pkgs/by-name/xe/xemu/package.nix

```diff
@@ -91,13 +92,18 @@ stdenv.mkDerivation (finalAttrs: {
     })
   ];
 
-  preConfigure = ''
+  postPatch = ''
     patchShebangs .
```

Comment:
Can we limit that to some directories?

Reply:
Well, I tried to list the scripts but it became unmanageable too fast.

I will review this later.